wiki/Version-4.0-Developer-Upgrade-Guide.md

```diff
@@ -537,7 +537,7 @@ The Ruleset class now respects sniff selection via `--sniffs=...`, even when in
 
 If your own test framework contained workarounds to get round the previous restriction, it should now be safe to remove those workarounds and to use the `--sniffs=...` argument when initiating the `Config` class.
 
-Typically, these type of workarounds can be found by searching for calls to the `Ruleset::registerSniffs()` method.
+Typically, these types of workarounds can be found by searching for calls to the `Ruleset::registerSniffs()` method.
```

Comment:
Not sure about this one. I believe singular is correct here.

Reply:
I'm not 100% sure as well, and thus I'm ok to leave it as is. Grammarly marks it as an agreement mistake, but it could be a false positive.

An alternative might be "Typically, these workarounds can be found by searching for calls to the `Ruleset::registerSniffs()` method." or even "Typically, these can be found by searching for calls to the `Ruleset::registerSniffs()` method."

Reply:
Ok, I reverted my change.